services/tilesets.js

```diff
@@ -16,6 +16,12 @@ var Tilesets = {};
  *
  * @param {Object} [config]
  * @param {string} [config.ownerId]
+ * @param {Object} [query]
+ * @param {string} [query.type] - Filter results by tileset type, either `raster` or `vector`.
```

Comment:
how about making the jsdoc an enum of values like https://github.com/mapbox/mapbox-sdk-js/blob/master/services/geocoding.js#L37 ?